docsrc/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
```

Comment:
I plan on using the sphinx_readable_theme 1.3.0 
https://sphinx-readable-theme.readthedocs.io 
```suggestion
#
import sphinx_readable_theme
```